test/sol5/kyberDao.js

```diff
@@ -146,17 +146,15 @@ contract('KyberDAO', function(accounts) {
         await deployContracts(20, currentBlock + 20, 8);
         await setupSimpleStakingData();
 
-        currentBlock = await Helper.getCurrentBlock();
-        await Helper.increaseBlockNumberBySendingEther(accounts[0], accounts[0], startBlock - currentBlock);
+        await time.advanceBlockTo(startBlock);
```

Comment:
can;t we call the helper function with this parameter?

Reply:
added a function named `increaseBlockNumberTo` which calls `time.advanceBlockTo`